content/guides/virtualizing/aws.md

```diff
@@ -123,7 +123,7 @@ Before proceeding any further, we will go over some of the parameters which you
 
 ##### Creating the AMI
 
-Now that we have uploaded our image, we can now proceed with making a snapshot.
+Now that we have uploaded our image, we can now proceed with making a snapshot:
 
 ```sh
 ec2-create-snapshot \
```

Comment:
Perhaps this was a typo? But for creating a snapshot with this command, it requires the `Volume ID` from the Elastic Block Store in EC2 after importing the volume rather than the bucket name.

Replace `haiku-folder` with `vol-XXXXXXXXXX` where the latter is the volume is used for the snapshot.